gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6-bin.zip
```

Comment:
Why is this only bumped to 5.6? The official ["library starter" bumps it to 6.4](https://github.com/enonic/starter-lib/blob/master/gradle/wrapper/gradle-wrapper.properties#L6) for XP7.